x-pack/plugins/security/server/authorization/check_privileges.test.ts

```diff
@@ -737,7 +737,7 @@ describe('#atSpaces', () => {
                 [`saved_object:${savedObjectTypes[0]}/get`]: false,
                 [`saved_object:${savedObjectTypes[1]}/get`]: true,
               },
-              // @ts-ignore this is wrong on purpose
+              // @ts-expect-error this is wrong on purpose
```

Comment:
sorry for the ping security team. File was modified then reverted, but until https://github.com/elastic/kibana/pull/70474 is merged, I can't revert this change.